src/Components/LandingPage/ExerciseListElement.tsx

```diff
@@ -1,4 +1,42 @@
-type Props = { exercise: string };
-export const ExerciseListElement = ({ exercise }: Props) => {
-  return <h1>{exercise}</h1>;
+import { ChartBar, Play } from 'phosphor-react';
+import { useNavigate } from 'react-router-dom';
+import { MouseEvent } from 'react';
+
+const EXERCISE: string = 'exercise';
+type Props = { title: string; description: string };
```

Comment:
```suggestion
const EXERCISE: string = 'exercise';

type Props = { title: string; description: string };
```
Jeg vil gerne have et white space imellem din EXERCISE konstant og typen Props